{"variants":[{"paths":["\/documentation\/mdots\/autherror\/errordescription"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError\/errorDescription"},"sections":[],"kind":"symbol","schemaVersion":{"minor":3,"patch":0,"major":0},"hierarchy":{"paths":[["doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots","doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError"]]},"primaryContentSections":[{"declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"kind":"text","text":": "},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"? { ","kind":"text"},{"text":"get","kind":"keyword"},{"kind":"text","text":" }"}]}],"kind":"declarations"},{"kind":"content","content":[{"anchor":"discussion","level":2,"type":"heading","text":"Discussion"},{"type":"paragraph","inlineContent":[{"type":"text","text":"This property provides a localized description for each error case, making it easier to"},{"text":" ","type":"text"},{"text":"understand what went wrong during the authentication process.","type":"text"}]}]}],"metadata":{"title":"errorDescription","role":"symbol","modules":[{"name":"MDots"}],"roleHeading":"Instance Property","symbolKind":"property","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":"?","kind":"text"}],"externalID":"s:5MDots9AuthErrorO16errorDescriptionSSSgvp"},"abstract":[{"type":"text","text":"A description of the error, suitable for displaying to the user."}],"references":{"doc://com.estelapasjak.Mdots/documentation/MDots/AuthError/errorDescription":{"role":"symbol","type":"topic","title":"errorDescription","identifier":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError\/errorDescription","kind":"symbol","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?"}],"abstract":[{"type":"text","text":"A description of the error, suitable for displaying to the user."}],"url":"\/documentation\/mdots\/autherror\/errordescription"},"doc://com.estelapasjak.Mdots/documentation/MDots/AuthError":{"title":"AuthError","url":"\/documentation\/mdots\/autherror","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"AuthError","kind":"identifier"}],"kind":"symbol","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"AuthError"}],"identifier":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError","type":"topic","abstract":[{"type":"text","text":"An enumeration representing various authentication errors that can occur."}]},"doc://com.estelapasjak.Mdots/documentation/MDots":{"type":"topic","role":"collection","url":"\/documentation\/mdots","identifier":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots","abstract":[],"images":[{"identifier":"appIcon.png","type":"icon"}],"kind":"symbol","title":"MDots"},"appIcon.png":{"alt":"The logo representing the MDot sensor application.","identifier":"appIcon.png","variants":[{"url":"\/images\/appIcon.png","traits":["1x","light"]}],"type":"image"}}}